src/openrct2/world/sprite.c

```diff
@@ -285,8 +261,39 @@ static void sprite_reset(rct_unk_sprite *sprite)
 
     sprite->linked_list_type_offset = llto;
     sprite->next = next;
+    sprite->next_in_quadrant = next_in_quadrant;
     sprite->previous = prev;
     sprite->sprite_index = sprite_index;
+    sprite->sprite_identifier = SPRITE_IDENTIFIER_NULL;
+}
+
+/**
+* Clears all the unused sprite memory to zero. Probably so that it can be compressed better when saving.
+*  rct2: 0x0069EBA4
+*/
+void sprite_clear_all_unused()
```

Comment:
If you think its too risky, then remove the reset version for this release and then merge them for next release. I don't see the need for both.